libs/core/src/lib/datetime-picker/datetime-picker.component.html

```diff
@@ -57,22 +53,37 @@
                     [aggregatedYearGrid]="aggregatedYearGrid"
                     [yearGrid]="yearGrid"
                     [startingDayOfWeek]="startingDayOfWeek"
+                    (activeViewChange)="handleCalendarActiveViewChange($event)"
+                    (selectedDateChange)="handleDateChange($event)"
+                    (isValidDateChange)="isInvalidDateInputHandler($event)"
                 ></fd-calendar>
                 <div class="fd-datetime__separator"></div>
                 <fd-time
                     [disabled]="disabled"
                     [keepTwoDigits]="keepTwoDigitsTime"
                     [meridian]="meridian"
-                    [ngModel]="time"
+                    [ngModel]="tempTime"
                     [compact]="compact"
                     [spinnerButtons]="spinnerButtons"
-                    (ngModelChange)="handleTimeChange($event)"
                     [displaySeconds]="displaySeconds"
                     [displayMinutes]="displayMinutes"
                     [displayHours]="displayHours"
+                    (ngModelChange)="handleTimeChange($event)"
                     (focusArrowLeft)="focusArrowLeft()"
                 ></fd-time>
             </div>
+            <div fd-popover-body-footer>
+                <div fd-bar [barDesign]="'footer'" [cosy]="!compact">
```

Comment:
Use static string binding if possible

Reply:
Should the Datetime Picker have the footer by default? Maybe we should make this optional?

Reply:
No specs for this but I suppose making it optional couldn't hurt